packages/lesswrong/server/migrations/meta/utils.ts

```diff
@@ -307,18 +309,24 @@ export const normalizeEditableField = async <N extends CollectionNameString>(
       "wordCount" = COALESCE((p."${fieldName}"->>'wordCount')::INTEGER, r."wordCount"),
       "updateType" = COALESCE(p."${fieldName}"->>'updateType', r."updateType"),
       "commitMessage" = COALESCE(p."${fieldName}"->>'commitMessage', r."commitMessage"),
-      "originalContents" = COALESCE(p."${fieldName}"->'originalContents', r."originalContents")
+      "originalContents" = COALESCE(p."${fieldName}"->'originalContents', r."originalContents"),
+      "draft" = COALESCE(p."draft", FALSE),
+      "collectionName" = $1
     FROM "${collectionName}" AS p
     WHERE
-      r."collectionName" = '${collectionName}'
+      r."collectionName" = $1
       AND r."fieldName" = '${fieldName}'
       AND p."${fieldName}_latest" = r."_id"
       AND p."${fieldName}"->>'html' <> r."html"
-  `);
+  `, [collectionName]);
 
   // eslint-disable-next-line no-console
   console.log("Dropping denormalized field...");
   await dropRemovedField(db, collection, fieldName);
+
+  // eslint-disable-next-line no-console
+  console.log("Undrafting most recent revisions for published posts...");
+  await undraftPublicPostRevisions(db);
```

Comment:
EAF has already run this migration so we'll have to trigger it manually instead, but it'll run automatically for LW

Reply:
Is this necessary in addition to the change above? To me it reads like they are doing the same thing

Reply:
Yeah - I'm just being overly cautious